PyInstaller/utils/win32/winmanifest.py

```diff
@@ -921,9 +927,11 @@ def toxml(self, encoding="UTF-8"):
         # version-encoding-standalone (standalone being optional), otherwise 
         # if it is embedded in an exe the exe will fail to launch! 
         # ('application configuration incorrect')
-        xmlstr = domtree.toxml(encoding).replace(
-            '<?xml version="1.0" encoding="%s"?>' % encoding, 
-            '<?xml version="1.0" encoding="%s" standalone="yes"?>' % encoding)
+        xmlstr = domtree.toxml(encoding)
+        xmlstr = xmlstr.decode().strip(os.linesep).replace(
```

Comment:
This was done because I was getting failures on PY3, and the sister method (toprettyxml), just above this one in the file, is structured this way. That is to say that this worked and matched what was already there. There was no other particularly good reason.

Looking more closely at toprettyxml(), I am wondering if it should be:

```
xmlstr = xmlstr.decode().strip(newl).replace(
```

and not 

```
xmlstr = xmlstr.decode().strip(os.linesep).replace(
```